services/user.py

```diff
@@ -0,0 +1,52 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(username: str,
+                password: str,
+                email: str = None,
+                first_name: str = None,
+                last_name: str = None) -> None:
+    user = get_user_model().objects.create_user(
+        username=username, password=password
+    )
+    user_data = {
+        "email": email,
+        "first_name": first_name,
+        "last_name": last_name
+    }
+
+    for field, data in user_data.items():

```

Comment:
why do u need to create a dict to use items method later ?)
try to use kwargs as arg for this func